src/renderer/Renderer.ts

```diff
@@ -180,8 +176,8 @@ export class Renderer {
                     projectionMatrix: this.projectionMatrix,
                     positions: o.geometry.verticesBuffer,
                     normals: o.geometry.normalsBuffer,
-                    colors: o.geometry.colorsBuffer,
                     indices: o.geometry.indicesBuffer,
+                    ...o.geometry.material,
```

Comment:
Could we have a named parameter here? All the other parameters are named, so seeing this made me a bit confused.

Reply:
This is the same as `materialColor: o.geometry.material.materialColor, materialShininess: o.geometry.material.materialShininess`. Should I expand to that full thing?

Reply:
I think it might be more explicit, but idk - up to you.

Reply:
For someone who knows nothing about the structure of `o.geometry.material`, they wouldn't know what this destructure turns into. They'd have to search for the definition of it to see. But again, that's just one lookup (and with typescript it's easy to see the types of things), so it's not a big deal. I'm fine with either.

Reply:
thats fine, I can add it